app/src/main/java/com/metrostateics499/vre_app/CreateContacts.kt

```diff
@@ -0,0 +1,109 @@
+package com.metrostateics499.vre_app
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.widget.*
+import androidx.appcompat.app.AppCompatActivity
+
+/**
+ * Create contacts
+ *
+ * @constructor Create empty Create contacts
+ */
+class CreateContacts : AppCompatActivity() {
+    private lateinit var btnNewAccount: Button
+    private lateinit var fullName: EditText
+    private lateinit var phoneNum: EditText
+    private lateinit var listContact: ListView
+    private lateinit var btnDelete: Button
+
+    // Create the ArrayList and ArrayAdapter
+    private lateinit var listNames: ArrayList<String>
+    private lateinit var listOfContact: ArrayList<Contacts>
+    private lateinit var createAdapter: ArrayAdapter<String>
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_create_contacts)
+
+        btnNewAccount = findViewById(R.id.btnAddContact)
+        fullName = findViewById(R.id.editTextPersonName)
+        phoneNum = findViewById(R.id.editTextPhone)
+        listContact = findViewById(R.id.contactList)
+        btnDelete = findViewById(R.id.btnDeleteContact)
+
+        listNames = ArrayList()
+        listOfContact = ArrayList()
+        createAdapter = ArrayAdapter(this, android.R.layout.simple_list_item_1, listNames)
+
+        listContact.adapter = createAdapter
+
+        btnNewAccount.setOnClickListener {
+            if (fullName.text.toString().isNotEmpty() && phoneNum.text.toString().isNotEmpty()) {
+                if (fullName.length() in 6..36 ) {
```

Comment:
let's change minimum from 6 to 2. User might want to only do first name and the shortest is 2 letters.
```suggestion
        fullName = findViewById(R.id.editTextPersonName)
        phoneNum = findViewById(R.id.editTextPhone)
        listContact = findViewById(R.id.contactList)
        btnDelete = findViewById(R.id.btnDeleteContact)

        listNames = ArrayList()
        listOfContact = ArrayList()
        createAdapter = ArrayAdapter(this, android.R.layout.simple_list_item_1, listNames)

        listContact.adapter = createAdapter

        btnNewAccount.setOnClickListener {
            if (fullName.text.toString().isNotEmpty() && phoneNum.text.toString().isNotEmpty()) {
                if (fullName.length() in 2..36 ) {
```